pkg/environment/environment.go

```diff
@@ -94,6 +94,8 @@ func newEnvironment(id string, properties map[string]string) (Environment, error
 }
 
 func NewEnvironment(id string, name string, group string, environmentUrl string, envTokenName string) Environment {
+	environmentUrl = strings.TrimSuffix(environmentUrl, "/")
```

Comment:
Cool, awesome! :1st_place_medal: 

One minor thing: Can you add a unit test for that in `pkg/environment/environment_test.go`?

Reply:
I need a badge. My first ever Go unit test. :) Hope that's all good! Merry Christmas.

Reply:
Cool, the test looks awesome! Thank you! :rocket: Merry Christmas and a Happy new Year! :christmas_tree: